ceph.spec.in

```diff
@@ -1086,36 +1081,19 @@ rm -rf %{buildroot}
 
 %post base
 /sbin/ldconfig
-%if 0%{?suse_version}
-%fillup_only
-if [ $1 -eq 1 ] ; then
-/usr/bin/systemctl preset ceph-disk@\*.service ceph.target >/dev/null 2>&1 || :
-fi
-%endif
-%if 0%{?fedora} || 0%{?rhel}
-%systemd_post ceph-disk@\*.service ceph.target
```

Comment:
We should not remove the systemd_post for ceph.target, here. The same goes for the suse part a bit up.